tpchgen/src/text.rs

```diff
@@ -41,25 +41,22 @@ impl TextPool {
 
     /// Returns a new text pool with a predefined size and set of distributions.
     pub fn new(size: i32, distributions: &Distributions) -> Self {
-        let mut output = ByteArrayBuilder::new(size as usize + Self::MAX_SENTENCE_LENGTH as usize);
         let mut rng = RowRandomInt::new(933588178, i32::MAX);
+        let mut text_bytes = Vec::with_capacity(size as usize + Self::MAX_SENTENCE_LENGTH as usize);
```

Comment:
`with_capacity` avoids (explicitly) zeroing out the memory before writing to it.

I didn't see much of difference on mac, but I think mac will zero out memory anyways. I'm curious to see if this would speed things up on linux.